packages/wdio-browserstack-service/src/insights-handler.ts

```diff
@@ -48,6 +65,13 @@ class _InsightsHandler {
             sessionId,
             product: isAppAutomate ? 'app-automate' : 'automate'
         }
+
+        this.registerListeners()
+    }
+
+    registerListeners() {
+        process.removeAllListeners(`bs:addLog:${process.pid}`)
+        process.on(`bs:addLog:${process.pid}`, this.appendTestItemLog.bind(this))
```

Comment:
similar to logPatcher, are we able to listen this process when multiple parallels are running ?

Reply:
Yup verified parallel case too